.github/workflows/centos7.yml

```diff
@@ -139,4 +161,10 @@ jobs:
       run: |
         gh release upload "$tag" _build/*.tar.gz _build/*.rpm
 
-  
\ No newline at end of file
+    - name: Cache vcpkg binary dir
+      if: always()
```

Comment:
Even in the case of build errors ?

Reply:
Good question :
I think it makes sense:
- if the dependencies build works and the project build fails, we need to keep the dependencies to not rebuild them in a later workflow
- if it's the build of a dependency which fails, I think it is still better to save the cache because other dependencies may have succeeded. Also, because we restore the previous cache, this should keep previously cached items which have not been rebuilt (like the ones that would have been rebuilt after the dependency that failed)